src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/initiation/InitiationTaskAttributesMapper.java

```diff
@@ -33,6 +33,36 @@ public InitiationTaskAttributesMapper(ObjectMapper objectMapper) {
 
     public Map<String, Object> mapTaskAttributes(CamundaTask camundaTask, Map<String, CamundaVariable> variables) {
         // Local Variables
+        String type = getType(camundaTask, variables);
+
+        Map<String, Object> attributes = new ConcurrentHashMap<>();
+        attributes.put(TASK_TYPE.value(), type);
+
+        setAttributesFromCamundaTask(camundaTask, attributes);
+
+        variables.entrySet().stream()
+            .filter(variable -> !variable.getKey().equals(DUE_DATE.value())) // done
```

Comment:
Please remove the comment at the end. Same below

Reply:
Fixed